docs/how-to/how-to-run-marain-locally.md

```diff
@@ -0,0 +1,110 @@
+# How to run Marain locally
+
+## Introduction
+
+Whilst it is preferable to develop against Marain instances hosted in the cloud, it will sometimes be necessary to run Marain locally for testing purposes. There are a couple of options here:
+1. All Marain services are run locally
+2. Only specific Marain services are run locally
+
+This document explains the first of these options. The other will be covered in a separate guide.
+
+## Overview
+
+To run any of the services locally, you need to do the following:
+1. Retrieve the code for the services you wish to run.
+2. Apply appropriate configuration for each service
+3. Run the service
+4. For multi-tenanted services, create the appropriate service registrations in your local instance of `Marain.Tenancy`.
+5. Create a client tenant to use when calling services.
+
+Since you'll be running multiple services, it's preferable to do so without needing an instance of Visual Studio open for each one. As long as you don't need them running in Debug mode, you can do this using the Azure Functions Core Tools (if you don't have them installed, do so now from npm - https://www.npmjs.com/package/azure-functions-core-tools).
```

Comment:
Consider adding instructions for how to update it if you've already got it, because this thing gets out of date. (And maybe point out that the copy that VS keeps up to date is a different one from this, so they can drift apart.)